templates/set-lcm-mode.ps1.erb

```diff
@@ -0,0 +1,51 @@
+Write-Host "Setting LCM"
+$LCMRefreshMode = '<%= @refresh_mode %>'
+$generationDate = (get-date -format "M/d/yyyy HH:mm:ss")
+$encoding       = [system.Text.Encoding]::UTF8
```

Comment:
Yeah, I tend to write code that same way - long form in scripts, except for a few exceptions.

* Omit `System.` since it's redundant and typically implied (which can help keep line length down)
* `%` instead of `ForEach-Object`
* `?` instead of `Where-Object`

I have a little style guide I wrote years ago around here somewhere.....